src/icons/folderIcons.ts

```diff
@@ -55,7 +55,7 @@ export const folderIcons: FolderTheme[] = [
           'pictures',
         ],
       },
-      { name: 'folder-scripts', folderNames: ['script', 'scripts'] },
+      { name: 'folder-scripts', folderNames: ['script', 'scripts', 'scripting'] },
```

Comment:
The pipeline is still complaining about the formatting, could you please fix that by running prettier?

Reply:
Sure

Reply:
Fixes